webofneeds/won-owner-webapp/src/main/webapp/app/service/won.js

```diff
@@ -1768,17 +1768,27 @@ WonMessage.prototype = {
     }
 
     if (innermostEnvelopes.length == 0) {
-      this.parseErrors.push("no innermost envelope found");

```

Comment:
instead of doing any of this please just print the graphUris as part of the console.warn within this file line#:918 that way we do not have to dabble with the stringified version of the graphUris `wonMessage.graphUris`